index.php

```diff
@@ -40,7 +40,54 @@
     $output = shell_exec($command);
     Echo  "<h2>Last Winner: ".$output."</h2>";
   ?>
+<script>
+window.onload = function () {
 
+var dataPoints = [];
+ 
+var chart = new CanvasJS.Chart("chartContainer", {
+  animationEnabled: true,
+  exportEnabled: true,
+  title:{
+    text: "League's Average Performance"
+  },
+  axisY: {
+    title: "Strokes"
+  },
+  data: [{
+    type: "column",
+    toolTipContent: "{y} strokes over",
+    dataPoints: dataPoints
+  }]
+});
+ 
+$.get("ScriptFiles/Averages.csv", getDataPointsFromCSV);
+ 
+//CSV Format
+//Year,Volume
+function getDataPointsFromCSV(csv) {
+  var csvLines = points = [];
+  csvLines = csv.split(/[\r?\n|\r|\n]+/);
+  for (var i = 0; i < csvLines.length; i++) {
+    if (csvLines[i].length > 0) {
+      points = csvLines[i].split(",");
+      dataPoints.push({
+        label: points[0],
+        y: parseFloat(points[1])
+      });
+    }
+  }
+  chart.render();
+}
+ 
+}
+</script>
+</head>
+<body>
+<div id="chartContainer" style="height: 370px; width: 100%;"></div>
+<script type="text/javascript" src="https://canvasjs.com/assets/script/jquery-1.11.1.min.js"></script>
+<script src="https://canvasjs.com/assets/script/canvasjs.min.js"></script>
+</body>
```

Comment:
To resolve sean's comment I would just take out the extra `</head>`, `<body>`, and `</body>` tags. I think you should be able to approve the change here, not sure it's a new feature I haven't used before.

```suggestion
</script>
<div id="chartContainer" style="height: 370px; width: 100%;"></div>
<script type="text/javascript" src="https://canvasjs.com/assets/script/jquery-1.11.1.min.js"></script>
<script src="https://canvasjs.com/assets/script/canvasjs.min.js"></script>
```